### PR TITLE
Update server-requirements.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2509
Added important note.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md [hunk 1]
```diff
@@ -83,6 +83,9 @@ In addition to the latest service pack, you'll want to ensure the following upda
 ## Back end databases that will work with Skype for Business Server 2015
 <a name="DBs"> </a>
 
+> [!IMPORTANT]
+> SQL back end must be local to the SFB front ends (not across a low speed link). It is not supported to share SQL back ends between two or more pools.
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md]:
```suggestion
> For a well-connected SQL back end, the connection to the SFB front end should be local, and not across a low speed link. Sharing SQL back ends between two or more pools is not supported.
```

File: Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md [hunk 2]
```diff
@@ -83,6 +83,9 @@ In addition to the latest service pack, you'll want to ensure the following upda
 ## Back end databases that will work with Skype for Business Server 2015
 <a name="DBs"> </a>
 
+> [!IMPORTANT]
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md]:
```suggestion
> [!NOTE]
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@LanaChin Hi Lana, any updates on this? PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
@scanum - I'm looking into this one. Thanks.

**Comment by LanaChin** [comment]
After consulting with Heidi, moved and separated the note into two bullet points.

**Pull request merged**